sample-app/screens/HomeScreen.js

```diff
@@ -227,12 +341,23 @@ export default class HomeScreen extends React.Component<Props, State> {
               onPress={() => this.makeGooglePayPayment(false)}
             />
           )}
+          <View style={styles.spacing} />
+          <NativePayButton
+            style={styles.payButtonStyle}
+            setThemeStyle={GooglePayButtonStyle.Dark}
```

Comment:
This looks wrong with this platform specific thing in there. I'd rather we handle the GooglePay and ApplePay buttons separately and not "merge" them to be a "NativePay" button.